src/components/Header/Header.js

```diff
@@ -33,6 +36,23 @@ export const Header = ({ showWalletPopup, walletInfo, currentTheme }) => {
 				<Network>
 					<NetworkLabel>{networkName}</NetworkLabel>
 				</Network>
+				<Switch>
+					<Route
+						path={ROUTES.Loans}
+						render={() => (
+							<StyledButton size="sm" disabled={true}>
+								{t('header.announcements.ether-collateral')}
+							</StyledButton>
+						)}
+					/>
+					<Route
+						render={() => (
+							<StyledButton size="sm" onClick={() => history.push(ROUTES.Loans)}>
+								{t('header.announcements.ether-collateral')}
+							</StyledButton>
+						)}
+					/>
+				</Switch>
```

Comment:
@clementbalestrat I think this is a nice pattern. I needed disable the button when I'm on the "Loans" page, and react-router path matching seems like the perfect solution for this.